src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -618,7 +618,7 @@ public void visitCtTypeParameter(CtTypeParameter typeParameter) {
 		} else {
 			printer.writeIdentifier(ref.getSimpleName());
 		}
-		if (ref.getBoundingType() != null) {
+		if (ref.getBoundingType() != null && !ref.getBoundingType().equals(ref.getFactory().Type().OBJECT)) { // we ignore printing "extends Object"
```

Comment:
one solution may be to simply remove this and always write "extends Object"

Reply:
I vote for shorter printed source code. So never print "... extends Object"

Reply:
> So never print "... extends Object"

And then I have a bug with assertj-core.